libs/types/src/locations.rs

```diff
@@ -99,12 +99,14 @@ mod test {
 
 	#[test]
 	fn from_xcm_versioned_address_works() {
-		let xa = VersionedMultiLocation::V1(MultiLocation::default());
+		let xa = VersionedMultiLocation::V3(MultiLocation::default());
 		let l = Location::from(xa.clone());
 		assert_eq!(
 			l,
 			Location::XCM(sp_core::H256(
 				<[u8; 32]>::from_hex(
+					// todo(nuno): failing, expecting
+					// "0x5a121beb1148b31fc56f3d26f80800fd9eb4a90435a72d3cc74c42bc72bca9b8"
```

Comment:
@wischli as we updated this to XCM v3, this test fails because the encoded value is not the expected, hardcoded value but the one I left in the comment instead. Just want to make sure it's safe for us to update the test and whether any migration or any other considerations need to be taken into account here?